<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HintPage</class>
 <widget class="QWidget" name="HintPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelHintNone">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioHintNone">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelHintMedium">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioHintMedium">
       <property name="text">
        <string>Medium</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="labelHintSlight">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioHintSlight">
       <property name="text">
        <string>Slight</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="labelHintFull">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioHintFull">
       <property name="text">
        <string>Full</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
